### ISSUE TITLE
[VMAgent  CSS] - DOWN no more highlighted in red

### ISSUE THREAD

**Issue by alsitn** [VictoriaMetrics/VictoriaMetrics#1461]
**Describe the bug**
as per subject, see image below.

**To Reproduce**
Open IP:8429/targets

**Expected behavior**
It was red and it should stay that way

**Logs**

**Screenshots**
![no-more-red](https://user-images.githubusercontent.com/2231287/125923228-d5b04582-4a64-4024-befd-c591b8d1dd56.png)

**Version**
```
$ ./vmagent-prod -version
vmagent-20210715-111034-tags-v1.63.0-0-g61cc13c16
```

**Comment by valyala** [comment]
@alsitn , the issue should be fixed in the commit a0e18f06ebbc30752a158c29f7ced24b61120bf7 . It will be included in the next release of VictoriaMetrics.

**Comment by valyala** [comment]
vmagent and single-node VictoriaMetrics should correctly highlight DOWN targets in red at `/targets` page starting from [v1.64.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
